internal/oci/oci_linux.go

```diff
@@ -127,6 +133,31 @@ func (r *runtimeOCI) containerStats(ctr *Container, cgroup string) (*ContainerSt
 	return stats, nil
 }
 
+// findSystemdCgroupPath takes a string in the form pod_abc-123.slice,
+// and returns the path under systemd it lives in.
+// this path will look like: /pod_abc.slice/pod_abc-123.slice
+func findSystemdCgroupPath(cgroup string) string {
```

Comment:
We can use ExpandSlice from libcontainer: https://github.com/opencontainers/runc/blob/master/libcontainer/cgroups/systemd/apply_systemd.go#L360

Reply:
I had a feeling this already existed :sweat_smile: